cmd/cli/main.go

```diff
@@ -2,39 +2,47 @@ package main
 
 import (
 	"flag"
-	"strings"
+	"log/slog"
```

Comment:
**question:**

I recall we wanted to do some evaluation of standard library `slog` vs. `zap` before making the switch (e.g. performance- and features-wise).

or are we changing just because `slog` is part of a standard go now?

Reply:
yes it's part of stdlib and i think we discussed it more broadly, using it in a bunch of projects and i saw pretty decent benchmarks (don't recall where). 

better to finally use standard lib tooling vs external packages.